src/consts/interfaces.consts.ts

```diff
@@ -167,7 +167,7 @@ export interface IPrepareDepositTxsResponse {
 	unsignedTxs: IPeanutUnsignedTransaction[]
 }
 
-export interface IPrepareXchainRequestFulfillmentTransactionProps {
+export interface IPrepareXchainRequestFulfillmentTransactionResponse {
```

Comment:
Q: this breaks peanut-ui right? Is there an accompanying PR?

Reply:
it does not because we dont import this by name anywhere